.travis.yml

```diff
@@ -122,6 +122,9 @@ before_script:
   - tar xzf tar-output/ledgersmb-travis.tar.gz
   - cd ledgersmb
   - cp conf/ledgersmb.conf.default ledgersmb.conf
+  - if [ "${DOJO_BUILT}" != "" ]; then
+      sed -r -i -e "s/^( |\t|#)*dojo_built( |\t)*=.*/dojo_built = ${DOJO_BUILT}/" ledgersmb.conf ;
+    fi
```

Comment:
They are out of sync. Maintaining them will always be a problem. I advocate using only one file and toggling the option.